pkg/blaker/blaker.go

```diff
@@ -45,15 +45,11 @@ func (b *Blaker) RunCmd(input *RunCmdInput) (cmd.Status, error) {
 		return cmd.Status{}, err
 	}
 	if breakTime != nil && b.clock.Now().After(*breakTime) {
-		msg := fmt.Sprintf("the command cannot be run after %s. skipped command: `%s %s`",
-			breakTime.Format(time.RFC3339),
-			input.Command,
-			strings.Join(input.Args, " "),
-		)
-		if _, err := fmt.Fprintf(input.Stderr, msg); err != nil {
-			return cmd.Status{}, errors.Wrapf(err, "failed to write skipped log: %s", msg)
+		skipErr := NewSkipError(*breakTime, input)
+		if _, err := fmt.Fprintf(input.Stderr, skipErr.Error()); err != nil {
+			return cmd.Status{}, errors.Wrapf(err, "failed to write skipped log: %s", skipErr.Error())
```

Comment:
I think it would be nicer not to write message here but in caller side.

Reply:
fix https://github.com/cynipe/blaker/pull/3/commits/6ca8f587a162e5170aacfa3843a5c1a1e6e9e288